src/components/Mlp/MlpSwap.js

```diff
@@ -89,6 +89,9 @@ function getStakingData(stakingInfo) {
     };
   }
 
+  // temp hardcode
+  data['stakedMlpTracker'].tokensPerInterval = ethers.utils.parseEther(1)
```

Comment:
![image](https://user-images.githubusercontent.com/14157626/204721931-2beecf7b-75bf-4b33-b8b7-d9f1abd9ba04.png)

might need to use a string here

```suggestion
  data['stakedMlpTracker'].tokensPerInterval = ethers.utils.parseEther('1')
```

Reply:
my local dev is cooked, keeps erroring out so this is firing from the hip